test/testHTMLFromMarkdown.js

```diff
@@ -101,9 +100,7 @@ const assertMd = async (md, html, secrets = {}) => {
   // check equality of the dom, but throw assertion based on strings to visualize difference.
   const act = new JSDOM(generated.response.body);
   const exp = new JSDOM(html);
-  if (!eq(act.window.document.body, exp.window.document.body)) {
```

Comment:
why this change? please keep it this way as it is easier to debug the differences if the tests fail.

Reply:
Hi Tobias,

thanks for the review! The change was necessary to make the tests pass again;
assertEquivalentNode has machinery built in to display the contents of the dom elements,
so extra logic in here is not required :)

assertEquals should be able to use the same machinery IMHO, I am currently designing a method to let it display DOM elements in a generic way.

Reply:
> built in to display the contents of the dom elements,

yes, but it's not nicely formatted. so the diff is very difficult to spot:

before:
![image](https://user-images.githubusercontent.com/917628/60939859-48f1f280-a314-11e9-9827-3066b498e023.png)

after:
![image](https://user-images.githubusercontent.com/917628/60939864-4ee7d380-a314-11e9-830b-daf5097bfacf.png)


anyways...that's a minor detail for now.